frontend/__tests__/components/chat/expandable-message.test.tsx

```diff
@@ -10,11 +10,25 @@ vi.mock("react-i18next", async () => {
   return {
     ...actual,
     useTranslation: () => ({
-      t: (key: string) => key,
+      t: (key: string) => {
+        // Return a different value for translation keys to verify translation is happening
+        if (key === "STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR") {
+          return "The request failed with an internal server error.";
+        }
+        if (key === "STATUS$ERROR_LLM_INTERNAL_SERVER_ERROR_MESSAGE") {
+          return "Something went wrong with the AI provider. This could be due to server overload or temporary issues. Please try again later.";
+        }
```

Comment:
You should just modify the translations for this!

Reply:
Oh this is a test file...

This file really shouldn't need to be edited I don't think...